server/routes/api/helpscout.ts

```diff
@@ -27,100 +29,38 @@ async function findUserByEmail(email: string) {
 		});
 	} else {
-		let affiliationName: string;
-		if (user.applicationBranch === "Mentor") {
-			affiliationName = "Affiliation";
-		} else {
-			affiliationName = "University";
-		}
-		let affiliationBlock: string;
-		let cellphoneBlock: string;
-		if (user.applicationData) {
-			affiliationBlock = createListBlock(affiliationName, findApplicationQuestion(user.applicationData, affiliationName.toLowerCase()));
-			cellphoneBlock = createListBlock("Cell Phone", findApplicationQuestion(user.applicationData, "phone-number"));
-
-		} else {
-			affiliationBlock = createListBlock(affiliationName, "");
-			cellphoneBlock = createListBlock("Cell Phone", "");
-		}
-
-		let appliedBlock: string;
-		if (user.applicationSubmitTime) {
-			appliedBlock = createListBlock("Applied?",
-				moment(user.applicationSubmitTime)
-					.format("DD-MMM-YYYY h:mm a"));
-		} else {
-			appliedBlock = createListBlock("Applied?",
-				"No application");
-		}
-
-		let acceptedText = badge("pending", "No decision");
-		let acceptedBlock = "";
-
-		let confirmationBlock = "";
-
-		if (user.confirmationBranch) {
-			acceptedText = user.accepted ? badge("success", "Accepted") : badge("error", "Rejected");
-			acceptedBlock = createListBlock("Accepted?", acceptedText);
-
-			let confirmedDate = "";
-			let confirmedColor = "pending";
-			if (user.accepted) {
-				confirmedColor = "success";
-				if (user.confirmationSubmitTime) {
-					confirmedDate = moment(user.confirmationSubmitTime)
-						.format("DD-MMM-YYYY h:mm a");
-				}
-			}
-
-			const confirmationText = user.confirmed ? `${badge(confirmedColor, "Confirmed")} ${confirmedDate}` : "No";
-			confirmationBlock = createListBlock("Confirmed?", confirmationText);
-		}
-
 		response.status(200).json({
-			"html": `
-				<h4>
-					<a href="mailto:${user.email}">${user.email}</a>
-				</h4>
-				<h4>${user.applicationBranch || "Has not applied"}</h4>
-				<ul class="c-sb-list c-sb-list--two-line">
-					${user.applicationData ? affiliationBlock : ""}
-					${user.applicationData ? cellphoneBlock : ""}
-					${appliedBlock}
-					${acceptedBlock}
-					${confirmationBlock}
-					<li class="c-sb-list-item">
-						<span class="c-sb-list-item__label">Ground Truth UUID<span class="c-sb-list-item__text">${user.uuid}</span>
-						</span>
-					</li>
-				</ul>
-			`.replace(/[\n\t]/g, "")
+			"html": MainHelpScoutTemplate.render({
+				name: user.name,
+				email: user.email,
+				uuid: user.uuid,
+				applicationSubmitTime: user.applicationSubmitTime ? moment(user.applicationSubmitTime)
+					.format("DD-MMM-YYYY h:mm a") : undefined,
+				applied: user.applied,
+				accepted: user.accepted,
+				confirmed: user.confirmed,
+				applicationBranch: user.applicationBranch,
+				confirmationBranch: user.confirmationBranch,
+				confirmationSubmitTime: user.confirmationSubmitTime ? moment(user.confirmationSubmitTime)
+					.format("DD-MMM-YYYY h:mm a") : undefined
+			}).replace(/[\r\n\t]/g, "")
```

Comment:
Any reason you're removing whitespace after rendering?

Reply:
Just that it looked weird in the terminal but it doesn't affect anything visually, so it's kinda superfluous